src/Gateway.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace DigiTickets\OmnipayOptomanyCheckout;
+
+use DigiTickets\OmnipayOptomanyCheckout\Message\AuthRequest;
+use DigiTickets\OmnipayOptomanyCheckout\Message\AuthResponse;
+use DigiTickets\OmnipayOptomanyCheckout\Message\CheckoutUrlRequest;
+use DigiTickets\OmnipayOptomanyCheckout\Message\CheckoutUrlResponse;
+use DigiTickets\OmnipayOptomanyCheckout\Message\CompletePurchaseRequest;
+use DigiTickets\OmnipayOptomanyCheckout\Message\PurchaseRequest;
+use DigiTickets\OmnipayOptomanyCheckout\Message\RefundRequest;
+use DNAPayments\DNAPayments;
+use Guzzle\Http\ClientInterface;
+use Omnipay\Common\AbstractGateway;
+use Omnipay\Common\Message\RequestInterface;
+use Symfony\Component\HttpFoundation\Request as HttpRequest;
+
+/**
+ * @method RequestInterface authorize(array $options = array())
```

Comment:
A lot of these functions aren't used. Pretty sure you don't need this block anyway.